index.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
 
     app.import('vendor/node-webkit/shim.js', { prepend: true });
     app.import({ development: 'vendor/node-webkit/reload.js' });
+    app.import({ development: '../../node_modules/ember-cli/node_modules/testem/node_modules/socket.io/node_modules/socket.io-client/socket.io.js' });
```

Comment:
There's actually another hidden issue here. Depending on the socket.io.js served off of the testem server causes issues.

This kind of falls to the same theme that it seems like the testem server mode was really designed to host the website content. It seems like testem assumes the browser will wait around until its ready to load the page (which makes sense, because a traditional browser does). But since we're loading the content locally, loading the socket.io.js file from testem via `<script>` tag may result in a dead link if the timing isn't right.

I don't see how to get around this unless we host index.html on testem, at which point changes how the app is ran all together .